README.md

```diff
@@ -1,102 +1,97 @@
-augur-core
-----------
+# augur-core
 
 [![Build Status](https://travis-ci.org/AugurProject/augur-core.svg)](https://travis-ci.org/AugurProject/augur-core)
 
-### Ethereum contracts for a decentralized prediction market platform
+Ethereum contracts for a decentralized prediction market platform.
 
-Depends on Serpent and Pyethereum
+## Installation
 
-# Installation
+You should already have a system-wide installation of Python and it should be
+Python 2.7.
 
-You should already have a system-wide installation of python, and it should be set to python-2.7.x.
+First install the dependencies, which include PyEthereum (the tool used to test
+Ethereum smart contracts from Python scripts) and the Serpent smart contract
+programming language:
 
-First set up pyethereum, which includes the tools used to test Ethereum smart contracts from python scripts, and the serpent smart contract programming language. <br>
-(can be done as your preferred user from your home directory)
-
-Pyethereum & Serpent Installation:
 ```
 sudo pip install -r requirements.txt
 ```
 
+(If installed outside of a Python virtualenv Serpent will be installed to
+`./src/serpent`. This isn't ideal but won't hurt anything.)
 
-Serpent can also be installed system-wide if available on your OS or distribution so you can do things like serpent compile and serpent mk_full_signature.<br>
-It is recommended to install the dev version of serpent directly from github as your user in your home directory as follows:
-
-Serpent Installation:
-```
-git clone https://github.com/ethereum/serpent.git
-cd serpent
-git checkout develop
-sudo pip install pytest
-sudo make && sudo make install
-sudo python setup.py install
-```
-
-Now we can try running some tests to make sure our installation worked.
+Now we can try running some tests to make sure our installation worked:
 
-Run some tests:
 ```
 cd tests
 python runtests.py
 ```
 
-
-# Docker
+## Docker
 
 You may run augur-core using docker as follows:
 
 ### Build:
+
 ```
 docker image build --tag augur-core-tests --file Dockerfile-test .
 ```
 
 ### Run:
+
 ```
-docker container rm -f augur-core-tests; docker container run --rm -it --name augur-core-tests augur-core-tests
+docker container rm -f augur-core-tests
```

Comment:
The `docker container run` command here explicitly provides a container name.  We could remove that, then it would create a new container each time with a random available name.  For the sake of this readme, that is probably the simpler way to go as we could then delete this line.  For my personal environment, I name the container and explicitly remove it so I don't have to worry about cruft buliding up over time, but this is really not something that should be addressed in this README.

So, to summarize, delete this line and change the `docker container run` command to not have a `--name` parameter.